themes/digital.gov/layouts/partials/core/get_related.html

```diff
@@ -1,5 +1,5 @@
 {{/* ======================================
-  Related Communities and Services
+  Related Communities
```

Comment:
Removed services from here since we are no longer hosting that info in this template